sebfcx/PROJET
# Add swiper slider to vehicules page

**Objective:** 

Adding a slider to display vehicules images on the `/vehicules` page.

**Modifications:** 

- I added Swiper cdn to the `vehicules.ejs` page
- I updated the vehicules.css file to update the size of the `.swiper` container so that it takes up 100% of its parents' height and width and removing unnecessary declarations
- I added the code for the Swiper slider to the `vehicules.ejs` page
- I removed unnecessary div containers from the `vehicules.ejs `page
- I updated the renderVehiculesPage method in the `mainController` (we will have to think about storing images in the database and retrieving them from there to be displayed on the website instead of what we are doing now)
- I updated the helmet middleware in the main `index.js` file to remove its content security policy which was preventing Swiper from displaying properly

**How to test:**

Download the branch locally, start the project and navigate to `/vehicules` to see the update.
